tests/verify_heap.cpp

```diff
@@ -107,34 +92,32 @@ static constexpr size_t heapInMB_default = 1024; // 1GB
  */
 auto main(int argc, char ** argv) -> int
 {
-    bool correct = false;
     bool machine_readable = false;
     size_t heapInMB = heapInMB_default;
     unsigned threads = threads_default;
     unsigned blocks = blocks_default;
 
     parse_cmdline(argc, argv, &heapInMB, &threads, &blocks, &machine_readable);
 
+    // TODO alpaka
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
     int computeCapabilityMajor = 0;
     cudaDeviceGetAttribute(
         &computeCapabilityMajor, cudaDevAttrComputeCapabilityMajor, 0);
     int computeCapabilityMinor = 0;
     cudaDeviceGetAttribute(
         &computeCapabilityMinor, cudaDevAttrComputeCapabilityMinor, 0);
-
     if(computeCapabilityMajor < 2)
     {
         std::cerr << "Error: Compute Capability >= 2.0 required. (is ";
         std::cerr << computeCapabilityMajor << "." << computeCapabilityMinor
                   << ")\n";
         return 1;
     }
+#endif
```

Comment:
Any way alpaka can check this? Otherwise: do we even need it?

Reply:
As I know alpaka is not providing this information. Note that ` if(computeCapabilityMajor < 2)` is not required since CUDA 9.0 compute architecture is at least 3.0

Reply:
Thx! I removed the check.